### PR TITLE
Fixing the error that the server name does not appear

### PR DESCRIPTION
**Pull request by NMW03**
image
image
image

### CODE REVIEW

File: Carl-bot/dist/metadata.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+  "author": {
+    "name": "NMW03",
+    "id": "534099893979971584"
+  },
+  "service": "Carl-bot",
+  "description": {
+    "en": "A fully customizable and modular discord bot featuring reaction roles, automod, logging, custom commands and much more."
+  },
+  "url": [
+    "carl.gg"
```
Comment (Alanexei) [inline Carl-bot/dist/metadata.json]: Make the url a string and not an array.

Comment (NMW03) [reply Carl-bot/dist/metadata.json]: oh, I'm was try to find where the error is

**Comment by doomlerd** [comment]
Resolve the conflicts and update your fork by the way.

**Comment by NMW03** [comment]
I don't understand why branch has conflicts.

**Comment by Bas950** [comment]
image
I understand

**Pull request closed without merging**